src/cms/static/js/pages/update_translation_state.js

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Just a note for discussion: until now we put all JS functions for one page in one file. Nothing wrong with splitting it into more files.

Reply:
I started separating the JS files for pages a while ago (c3b1153d1775bf75c1978020e3358faf886bb45a), because the single file was a bit overloaded and some functions are required on multiple templates, others only on individual ones.
I think it makes sense to have one JS-file per "functionality", even if this means many small files...

Reply:
Alright then, I'll try to remember to do this myself :+1: